linera-storage/src/lib.rs

```diff
@@ -473,4 +474,12 @@ where
     async fn get_blob(&self, blob_id: BlobId) -> Result<Blob, ExecutionError> {
         Ok(self.storage.read_blob(blob_id).await?)
     }
+
+    async fn assert_blob_exists(&self, blob_id: BlobId) -> Result<(), ExecutionError> {
+        ensure!(
+            self.storage.contains_blob(blob_id).await?,
+            ExecutionError::ViewError(ViewError::not_found("value for blob ID", blob_id))
+        );
+        Ok(())
+    }
```

Comment:
I guess we can remove this function now, and use `contains_blob` instead?

Reply:
The problem is how do I access the storage from `ExecutionStateView` if not through this path? I couldn't see a better way 🤔

Reply:
Ah, right. Then maybe change this into `contains_blob`, too?